GOVERNANCE.md

```diff
@@ -0,0 +1,135 @@
+### io.js Build Team Working Group
+
+The io.js build team is jointly governed by a Working Group (WG)
+which is responsible for high-level guidance of the project.
+
+The WG has final authority over this project including:
+
+* Technical direction
+* Project governance and process (including this policy)
+* Contribution policy
+* GitHub repository hosting
+* Conduct guidelines
+* Maintaining the list of additional Collaborators
+
+For the current list of WG members, see the project
+[README.md](./README.md#current-project-team-members).
+
+### Collaborators
+
+The [iojs/build](https://github.com/iojs/build),
+[iojs/docker-iojs](https://github.com/iojs/docker-iojs), and the
+[iojs/build-containers](https://github.com/iojs/build-containers)
+GitHub repositories are maintained by the WG and additional Collaborators
+who are added by the WG on an ongoing basis.
+
+Individuals making significant and valuable contributions are made
+Collaborators and given commit-access to the project. These
+individuals are identified by the WG and their addition as
+Collaborators is discussed during the weekly WG meeting.
```

Comment:
just catching up here but monthly sounds good to me too, I'd like to try and lock in a specific day/time for that, atm we have 10pm UTC on a Wednesday that's probably not going to work for @bnoordhuis at least: http://www.timeanddate.com/worldclock/fixedtime.html?msg=io.js+Build+WG+Meeting+2015-04-15&iso=20150415T22